packages/nuxt/src/module.ts

```
@@ -85,3 +100,53 @@
+        content: [
+          ${JSON.stringify(contentPath)}
+        ],
+        plugins: [
+          require('uploadthing/tw').uploadthingPlugin
+        ]
+      }
+    `,
+  });
+
+  // @ts-expect-error - Help pls
+  const twModuleOptions = (nuxt.options.tailwindcss ??= {}) as {
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)